apps/backoffice-v2/src/lib/blocks/hooks/useProcessTrackerBlock/useProcessTrackerBlock.ts

```diff
@@ -0,0 +1,23 @@
+import { createBlocksTyped } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
+import { useEntityLogic } from '@/pages/Entity/hooks/useEntityLogic/useEntityLogic';
+import { useMemo } from 'react';
+
+export const useProcessTrackerBlock = () => {
+  const { workflow, plugins } = useEntityLogic();
```

Comment:
Hooks with a suffix of `Logic` are meant to be consumed by their accompanied component and in this case `Entity.page.tsx`. If you need a subset of the logic please abstract just that logic and use that here instead.